values.yaml

```diff
@@ -77,3 +77,69 @@ storage:
   elasticsearch:
     port: 9200
     host: 127.0.0.1
+
+exporter:
+  enabled: false
+  image:
+    repository: registry.ng.bluemix.net/secadvisor_dev/skydive-flow-exporter
+    tag: 12
+  store:
+    bucket: "default"
+    objectPrefix: "default"
+    buffered:
+      max_flows_per_object: 60000
+  write:
+    s3:
+      endpoint: "http://localhost:9000"
+      region: "default"
+      use_api_key: true
+      api_key: ""
+      access_key: ""
+      secret_key: ""
+  env:
+    # For debug purposes
+    - name: SKYDIVE_LOGGING_LEVEL
+      value: "INFO"
+
+    # add this to use an api key for object store authentication
+    #- name:SKYDIVE_PIPELINE_WRITE_S3_API_KEY
+    #  value:""
+
+    # Maximum number of flows per object
+    - name: SKYDIVE_PIPELINE_STORE_BUFFERED_MAX_FLOWS_PER_OBJECT
+      value: "60000"
+
+    # Maximum time period for object
+    - name: SKYDIVE_PIPELINE_STORE_BUFFERED_MAX_SECONDS_PER_OBJECT
+      value: "60"
+
+    - name: SKYDIVE_PIPELINE_STORE_BUFFERED_MAX_FLOW_ARRAY_SIZE
+      value: "100000"
+
+    # New stream (folder) is opened every such period
+    - name: SKYDIVE_PIPELINE_STORE_BUFFERED_MAX_SECONDS_PER_STREAM
+      value: "86400"
+
+    # the rate in which the agents transfer flow updates to the analyzer
+    # Flows to be updated every 30 seconds (thus, every 60 seconds to defender)
+    - name: SKYDIVE_FLOW_UPDATE
+      value: "30"
+
+    # ANALYZER_STARTUP_CAPTURE_GREMLIN invoke at startup capturing in selected nodes (sets automatic capture on all but he loopback nodes)
+    - name: SKYDIVE_ANALYZER_STARTUP_CAPTURE_GREMLIN
+      value: "G.V().has('Name', NE('lo'))"
+
+    #capture all but skydive (8082) and port 9000
```

Comment:
I don't think filtering out port 9000 should be the default. If you upload the flows to S3 or COS traffic is in port 443 anyway. Security Advisor might miss some anomalous traffic to port 9000.